lib/notifications/receive.dart

```diff
@@ -149,8 +149,7 @@ class NotificationService {
         await addFcmToken(connection, token: token);
 
       case TargetPlatform.iOS:
-        const appBundleId = 'com.zulip.flutter'; // TODO(#407) find actual value live
-        await addApnsToken(connection, token: token, appid: appBundleId);
+        await addApnsToken(connection, token: token, appid: (await ZulipBinding.instance.packageInfo)?.packageName?? 'com.zulip.flutter');
```

Comment:
Bumping https://github.com/zulip/zulip-flutter/pull/1143#discussion_r1896163485

Let's include a `assert(debugLog(…))` here with a `TODO(log)` comment.

It would be helpful to structure this with an if-statement first.
```suggestion
        final packageInfo = await ZulipBinding.instance.packageInfo;
        if (packageInfo == null) {
          assert(debugLog('missing packageInfo')); // TODO(log)
        }
        await addApnsToken(connection,
          token: token, appid: packageInfo?.packageName ?? 'com.zulip.flutter');
```

Reply:
makes sense! I was supposed to think like that at first😅

Reply:
can we write this(assert(packageInfo == null, debugLog('missing packageInfo')) instead of an if statement ?